org.civicrm.reqangular/src/common/models/instances/session-instance.instance.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-env amd */
+
+define([
+  'common/modules/models-instances',
+  'common/models/instances/instance'
+], function (instances) {
+  'use strict';
+
+  instances.factory('SessionInstance', ['ModelInstance',
```

Comment:
do we really need an instance in this case? the model simply returns a "static" object with some data in it